packages/dialogs/src/RPA/Dialogs/library.py

```diff
@@ -696,6 +697,59 @@ def add_drop_down(
 
         self.add_element(element)
 
+    @keyword("Add Date Input", tags=["input"])
+    def add_date_input(
+        self,
+        name: str,
+        default: Optional[str] = None,
+        label: Optional[str] = None,
```

Comment:
I would just drop this from the public API if it isn't supported now. Optional arguments can always be added later.